### PR TITLE
misc: CI/Project changes

### PR DESCRIPTION
**Pull request by daudix-UFO**
# Description

Some changes in the CI and Project

## Type of change

- [ ] Bugfix (Change which fixes a issue)
- [ ] New feature (Change which adds new functionality)
- [x] Enhancement (Change which slightly improves existing code)
- [ ] Breaking change (This change will introduce incompatibility with existing functionality)

## Changelog 

- Removed hardcoded `anyascii` version as latest version is used automatically during build
- Removed `SECURITY.md` as this is unnecessary for Gradience
- Renamed CI back to CI
- Some wording and style changes

## Testing

- [x] I have tested my changes and verified that they work as expected 

### How to test the changes

No information provided.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,8 +14,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-
-name: Build
```
Comment (tfuxu) [inline .github/workflows/ci.yml:18]: `Build` looks better, and we essentially do only that in this workflow.

File: README.md [hunk 2]
```diff
@@ -144,7 +145,7 @@ dnf install gradience
 > **Warning**
 > Not available yet.
 
-### AUR 
+### Arch (AUR)
```
Comment (tfuxu) [inline README.md:148]: Maybe `Arch Linux (AUR)`?

File: build-aux/flatpak/com.github.GradienceTeam.Gradience.Devel.json [hunk 3]
```diff
@@ -106,8 +106,7 @@
             "buildsystem": "meson",
             "config-opts": [
                 "-Ddocs=false",
-                "-Dvapi=false",
-                "-Dbackend-gtk4=enabled"
+                "-Dbackends=gtk4"
```
Comment (tfuxu) [inline build-aux/flatpak/com.github.GradienceTeam.Gradience.Devel.json:109]: I told you before that this needs to remain unchanged (commit 4c5b07d40e28a7392b9d33546b9e3dd3a5966490)

File: build-aux/flatpak/com.github.GradienceTeam.Gradience.Devel.json [hunk 4]
```diff
@@ -106,8 +106,7 @@
             "buildsystem": "meson",
             "config-opts": [
                 "-Ddocs=false",
-                "-Dvapi=false",
```
Comment (tfuxu) [inline build-aux/flatpak/com.github.GradienceTeam.Gradience.Devel.json:109]: We don't need Vala libraries in Python project

File: build-aux/flatpak/com.github.GradienceTeam.Gradience.json [hunk 5]
```diff
@@ -106,7 +106,6 @@
             "buildsystem": "meson",
             "config-opts": [
                 "-Ddocs=false",
-                "-Dvapi=false",
```
Comment (tfuxu) [inline build-aux/flatpak/com.github.GradienceTeam.Gradience.json:109]: We don't need Vala libraries in Python project

File: HACKING.md [hunk 6]
```diff
@@ -99,4 +100,4 @@ ninja -C builddir run
 ```
 
 > **Note** 
-> During testing and development, as a convenience, you can use the `local.sh` script to quickly rebuild local builds.
+> During testing and development, as a convenience, you can use the `local.sh` script to quickly rebuild local builds.
```
Comment (tfuxu) [inline HACKING.md:103]: This looks like LF to CRLF EoL sequence change. We only use LF sequencing in this project

File: README.md [hunk 7]
```diff
@@ -250,4 +251,4 @@ Special thanks to:
 - Original author of Gradience, [Artyom Fomin](https://github.com/ArtyIF) for creating this project
 - [Weblate](https://weblate.org) for providing translation platform
 
-This README is based on README from [Kooha](https://github.com/SeaDve/Kooha) by [Dave Patrick Caberto](https://github.com/SeaDve)
+This README is based on README from [Kooha](https://github.com/SeaDve/Kooha) by [Dave Patrick Caberto](https://github.com/SeaDve)
```
Comment (tfuxu) [inline README.md:254]: This looks like LF to CRLF EoL sequence change. We only use LF sequencing in this project

File: ROADMAP.md [hunk 8]
```diff
@@ -24,10 +24,10 @@ This tool is fully usable, if you want a new feature, open an issue or submit a
...
 
-# Plugins
+### Plugins
 
 - [x] Add plugin support. Will help integration with others tools.
 - [x] Customize Firefox GNOME theme
 - [ ] Customize GNOME Shell **(High priority)**
 - [ ] Customize GDM **(High priority)**
-- [ ] Customize KvLibadwaita
+- [ ] Customize KvLibadwaita
```
Comment (tfuxu) [inline ROADMAP.md:33]: This looks like LF to CRLF EoL sequence change. We only use LF sequencing in this project

File: gradience/backend/struct.md [hunk 9]
```diff
@@ -2,4 +2,4 @@
 
 - `models/`  - objects containing various types of data, with logic to manipulate them
 - `theming/` - theme generation/manipulation specific modules
-- `utils/`   - general purpose utility modules
+- `utils/`   - general purpose utility modules
```
Comment (tfuxu) [inline gradience/backend/struct.md:5]: This looks like LF to CRLF EoL sequence change. We only use LF sequencing in this project

File: gradience/frontend/struct.md [hunk 10]
```diff
@@ -4,4 +4,4 @@
 - `dialogs/` - message boxes and simple popup windows
 - `utils/`   - general purpose utility modules and helpers for UI related stuff
 - `views/`   - more extensive widgets (eg. widget groups, lists) and windows
-- `widgets/` - general purpose graphical widgets
+- `widgets/` - general purpose graphical widgets
```
Comment (tfuxu) [inline gradience/frontend/struct.md:7]: This looks like LF to CRLF EoL sequence change. We only use LF sequencing in this project

File: SECURITY.md [hunk 11]
```diff
@@ -1,18 +0,0 @@
...
-| 0.3.x   | :white_check_mark: |
-| 0.2.x   | :white_check_mark: |
-| < 0.1   | :x:                |
-
-## Reporting a Vulnerability
-
-Normally Gradience can't have security issues because it's only generating CSS and JSON.
-
-BUT If you found a vulnerability please don't submit a public issue.
-Report this vulnerability to AdwCustomizerTeam@proton.me and include as much details as possible.
```
Comment (tfuxu) [inline SECURITY.md:18]: Why removing it tho?

**Review by tfuxu — CHANGES_REQUESTED** [review]
There are a couple of mistakes that need to be addressed. I don't understand why you removed `SECURITY.md` and you definitely need to revert those changes in Flatpak manifests.

File: SECURITY.md (hunk 11 above)
Comment (daudix-UFO) [reply SECURITY.md:18]: Do we have something that can have security issue?

Comment (tfuxu) [reply SECURITY.md:18]: Any amount of code can contain a security issue. As a recent quick example: https://github.com/bottlesdevs/Bottles/pull/2321

**Pull request closed without merging**